.github/workflows/ci.yaml

```diff
@@ -10,7 +10,7 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        os: [ubuntu-latest, macos-12, macos-14, windows-latest, ubuntu-arm]
+        os: [ubuntu-latest, macos-latest, windows-latest, ubuntu-arm]
```

Comment:
Thanks! Let's keep `macos-14` and just swap `macos-12` for `macos-latest` (which should be 15 nowadays).